test/test_data.py

```diff
@@ -275,6 +276,11 @@ def test_all(parsers=None, modules=None, terse=False, silent=True, summary=True,
 
 if __name__ == "__main__":
 
+    if "--debug" in sys.argv:
+        logging.getLogger().setLevel(logging.DEBUG)
+    else:
+        logging.getLogger().setLevel(logging.FATAL)
```

Comment:
Any reason for `FATAL` instead of `ERROR`? Just to suppress *all* output from the parsers/tests?

Also, went looking for description of `FATAL` in the docs, and was surprised to find that it's not mentioned even though it's identical to `CRITICAL`. See https://www.python.org/dev/peps/pep-0282/#levels.

Reply:
You're right. Changed to ERROR.